CI: purge man-db

ikerexxe opened the pull request:
This accelerates the CI.

Resolves: https://github.com/shadow-maint/shadow/issues/1240
Link: <https://101010.pl/@nabijaczleweli/114149412203886808>
Reported-by: nabijaczleweli <nabijaczleweli@nabijaczleweli.xyz>

alejandro-colomar reviewed (approved):
Wow!

alejandro-colomar commented:
> This accelerates the CI.
> 
> Resolves: #1240 Link: https://101010.pl/@nabijaczleweli/114149412203886808 Reported-by: nabijaczleweli [nabijaczleweli@nabijaczleweli.xyz](mailto:nabijaczleweli@nabijaczleweli.xyz)

I think `Closes:` is more common than `Resolves:` in the projects I've seen.

Here's the Linux kernel:

```
alx@devuan:~/src/linux/linux/master$ git log --grep '^\(Resolves\|Closes\):' | grep -o '\(Resolves\|Closes\):' | head -n1000 | sort | uniq -c;
   1000 Closes:
```

Shadow:

```
alx@devuan:~/src/shadow/shadow/master$ git log --grep '^\(Resolves\|Closes\):' | grep -o '\(Resolves\|Closes\):' | head -n1000 | sort | uniq -c
     44 Closes:
     42 Resolves:
```

BTW, @nabijaczleweli usually signs as `наб <nabijaczleweli@nabijaczleweli.xyz>` in Cyrillic.

ikerexxe commented:
> I think `Closes:` is more common than `Resolves:` in the projects I've seen.
> BTW, @nabijaczleweli usually signs as `наб <nabijaczleweli@nabijaczleweli.xyz>`, in Cyrillic.

I updated those two and with your permission I have also added `Reviewed-by: Alejandro Colomar <alx@kernel.org>`

alejandro-colomar commented:
Thanks!  I'll merge.
